subsys/net/l2/wifi/wifi_shell.c

```diff
@@ -1678,6 +1681,23 @@ static int wifi_ap_config_args_to_params(const struct shell *sh, size_t argc, ch
 			params->max_num_sta = (uint32_t)val;
 			params->type |= WIFI_AP_CONFIG_PARAM_MAX_NUM_STA;
 			break;
+		case 'I':
+			params->if_index = (uint8_t)atoi(optarg);
```

Comment:
We are trying to migrate away from `atoi` calls as it does not check errors. There are still atoi calls in the shell but these will need to be converted in the future. Please use `shell_strtol()` call instead here.

Reply:
Updated.